Products/CMFEditions/skins/CMFEditions/versions_history_form.pt

```diff
@@ -55,7 +55,7 @@
                     tal:condition="isModified">
                   <td class="VersionId"
                       tal:define="id current/version_id;">
-                    <span i18n:translate="">Working Copy</span>
+                    <span i18n:translate="">Current version</span>
```

Comment:
Since this is a message id, can we avoid retranlating it by just changing the (default) translation, like this:
<span i18n:translate="Working Copy">Current version</span>